lib/provider/camunda/parts/implementation/Callable.js

```diff
@@ -234,7 +234,7 @@ module.exports = function(element, bpmnFactory, options, translate) {
     validate: function(element, values, node) {
       var elementRef = values.callableElementRef;
       var type = getCallableType(element);
-      return isSupportedCallableType(type) && !elementRef ? { callableElementRef: 'Must provide a value.' } : {};
+      return isSupportedCallableType(type) && !elementRef ? { callableElementRef: translate('Must provide a value') + '.' } : {};
```

Comment:
```suggestion
      return isSupportedCallableType(type) && !elementRef ? { callableElementRef: translate('Must provide a value.') } : {};
```
Let's not exclude any parts of template strings. Some languages use own fullstop instead of common one.